src/test/regress/expected/rpt_joins.out

```diff
@@ -2153,10 +2153,49 @@ select max(c1) from pg_class left join t_5628 on true;
    2
 (1 row)
 
+--
+-- regression test for nest loop join of indexed rpt and entry, segment should not fall with segfault; see issue #12228
+--
+create table test_tz(tz interval) distributed replicated;
+insert into test_tz
+select i * '1 hour'::interval
+from generate_series(0,23) i;
+create index test_tz_idx on test_tz(tz);
+set optimizer=off;
+set enable_hashjoin=off;
+set enable_seqscan=off;
```

Comment:
IMO it makes sense to print original optimal plan without motion above inner subpath.

Reply:
My bad, the line `set enable_seqscan=off` was used to force index scan in inner subplan without calling `analyze` on table. I withdraw my point above.